source/s3_auto_ranged_put.c

```diff
@@ -605,40 +503,13 @@ static int s_s3_auto_ranged_put_header_block_done(
         s_s3_auto_ranged_put_unlock_synced_data(auto_ranged_put);
 
         return result;
-    } else if (request->desc_data.request_tag == AWS_S3_AUTO_RANGED_PUT_REQUEST_TAG_COMPLETE_MULTIPART_UPLOAD) {
-        aws_byte_buf_reset(&request->send_data.part_buffer->buffer, false);
+    } else if (request->request_tag == AWS_S3_AUTO_RANGED_PUT_REQUEST_TAG_COMPLETE_MULTIPART_UPLOAD) {
         return AWS_OP_SUCCESS;
     }
 
     return AWS_OP_SUCCESS;
 }
 
-static int s_s3_auto_ranged_put_incoming_body(
```

Comment:
Appending response body data to the correct buffer is now handled in the base Meta Request type.